build-config/resources/Ballerina.toml

```
@@ -54,6 +54,9 @@ path = "./lib/netty-handler-proxy-@netty.version@.jar"
 [[platform.java11.dependency]]
 path = "./lib/netty-codec-http2-@netty.version@.jar"
 
+[[platform.java11.dependency]]
+path = "./lib/netty-transport-native-unix-common-@netty.version@.jar"
```

Can we add the groupId and artifactId to this new dependency? I'll send another PR to update others
```suggestion
groupId = "io.netty"
artifactId = "netty-transport-native-unix-common"
version = "@netty.version@"
path = "./lib/netty-transport-native-unix-common-@netty.version@.jar"
```